browser/main/NoteList/index.js

```diff
@@ -349,6 +353,71 @@ class NoteList extends React.Component {
     })
   }
 
+  getSearchNotes () {
```

Comment:
To begin with my review, I'd like you to move this method to a module such as `lib/search` or something. But it's ok, I'll start my review :)